views/library/unified_search_list.html

```
@@ -1,154 +1,116 @@
+        </span>
+      </span>
+
+      <!-- Duration (tracks) -->
+      {{if and (eq .Type "track") .Duration}}
+      <span class="flex-shrink-0 text-xs text-gray-400 dark:text-gray-500 tabular-nums">{{duration .Duration}}</span>
+      {{end}}
+
+      <!-- Actions -->
+      <span class="flex-shrink-0 flex items-center gap-1">
+        {{if eq .Type "track"}}
+        <button class="text-gray-500 hover:text-gray-700 dark:text-gray-400 dark:hover:text-gray-200 w-7 h-7 flex items-center justify-center rounded-md hover:bg-gray-200/70 dark:hover:bg-neutral-700"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)